exercises/rectangles/tests/rectangles.rs

```diff
@@ -36,64 +36,42 @@ fn test_one_rectangle() {
 #[test]
 #[ignore]
 fn test_two_rectangles_no_shared_parts() {
-    let lines = vec![
-        "  +-+",
-        "  | |",
-        "+-+-+",
-        "| |  ",
-        "+-+  "
-        ];
+    let lines = vec!["  +-+", "  | |", "+-+-+", "| |  ", "+-+  "];
```

Comment:
Yeah, I was digging into that. But I'm not that excited about putting rustfmt directives throughout the codebase. I think our best approach may be to leave things as they are.